README.md

```diff
@@ -1,2 +1,73 @@
-# CORE
 [![Build Status](https://travis-ci.org/mentorpaired/core.svg?branch=staging)](https://travis-ci.org/mentorpaired/core) [![Coverage Status](https://coveralls.io/repos/github/mentorpaired/core/badge.svg?branch=staging)](https://coveralls.io/github/mentorpaired/core?branch=staging)
+
+# CORE
+Core is the backend of MentorPaired, an app which connects software engineering mentors to mentees who seek to learn any programming language.
+
+### Tech
+Core is written in Python3 and Django 3.0.3.
+
+### Installation For Ubuntu 18.04 Users
```

Comment:
This can be:
### Installation 
**Ubuntu 18.04 Users**

Reply:
Okay